src/Silverback.Integration.Kafka/Messaging/Broker/KafkaProducer.cs

```diff
@@ -7,17 +7,20 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 using Silverback.Messaging.Messages;
+using Silverback.Messaging.Publishing;
 using Silverback.Util;
 
 namespace Silverback.Messaging.Broker
 {
     public class KafkaProducer : Producer<KafkaBroker, KafkaProducerEndpoint>, IDisposable
     {
-        internal const string MessageKeyHeaderKey = "x-kafka-message-key";
+        internal const string PartitioningKeyHeaderKey = "x-kafka-partitioning-key";
```

Comment:
I suppose something went wrong with the rebase because I had recently changed this from `x-kafka-partitioning-key` to `x-kafka-message-key` and it's being reverted.

Reply:
Better that way?